examples/Makefile

```diff
@@ -1,41 +1,26 @@
-JULIAHOME := $(abspath ..)
+SRCDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+BUILDDIR := .
+JULIAHOME := $(abspath $(SRCDIR)/..)
+prefix := $(JULIAHOME)/usr
```

Comment:
this isn't the install prefix, this is the default build prefix. using build_bindir and build_libexecdir would be more correct, I think